src/core/dfn-panel.js

```diff
@@ -45,8 +45,13 @@ function createPanel(dfn) {
   const panel = html`
     <aside class="dfn-panel" id="${panelId}" hidden>
       <span class="caret"></span>
-      <b><a class="self-link" href="${href}">Permalink</a></b>
-      <b>Referenced in:</b>
+      <div>
+        <a class="self-link" href="${href}">Permalink</a>
+        ${dfn.matches("dfn[data-export]")
+          ? html`<span class="dfn-exported">exported</span>`
```

Comment:
Would this be helpful? 

```suggestion
          ? html`<span class="dfn-exported" title="Definition can be referenced by other specifications">exported</span>`
```